plugin/plugins/wasmipld/spec.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "WAC Specification"
```

Comment:
what is 'wac'?

Reply:
This will get extracted (and maybe a better name?) and made into an spec PR to the IPLD repo, but basically I created a new IPLD codec called WAC (webassembly codec). The name isn't really great, just named after the first use case I had in mind.

The idea is basically to have a simple codec that fits the IPLD data model, which both:
a) Gives us something to discuss about what the IPLD data model is
b) Means I can get data in/out of WASM losslessly without requiring tons of calls to extract each integer, string, byte array, etc. manually. I can copy entire nested maps of nested maps of lists and it's fine

Reply:
https://github.com/ipld/ipld/pull/226